Makefile

```diff
@@ -264,12 +264,23 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
 $(GOLANGCI_LINT): $(LOCALBIN)
 	test -s $(GOLANGCI_LINT) || { curl -sSfL $(GOLANGCI_LINT_INSTALL_SCRIPT) | bash -s $(GOLANGCI_LINT_VERSION); }
 
-CRD_REF_DOCS_DL_URL ?= 'https://github.com/elastic/crd-ref-docs/releases/download/v$(CRD_REF_DOCS_VERSION)/crd-ref-docs'
+
+ifeq ($(shell go env GOOS), linux)
+ARCH=x86_64
+else
+ARCH=$(strip $(shell go env GOARCH))
+endif
```

Comment:
Their OS and ARCH values are `uname -s` and `uname -m`

Reply:
i rewrite this whole shabnag